extra/material.js

```diff
@@ -2457,12 +2457,6 @@ function MaterialTab(tab, ctx) {
         }
         tab.addEventListener('click', function (e) {
             e.preventDefault();
-            var href = tab.href.split('#')[1];
-            var panel = ctx.element_.querySelector('#' + href);
-            ctx.resetTabState_();
-            ctx.resetPanelState_();
-            tab.classList.add(ctx.CssClasses_.ACTIVE_CLASS);
-            panel.classList.add(ctx.CssClasses_.ACTIVE_CLASS);
```

Comment:
Hmm, would mdl want this changed in their branch? It will currently fail if there is no dom panel for mdl to make visible, and if you are using pure mdl, this is probably an error that you want to see and then correct.
Like this person's issue which you have commented on before: google/material-design-lite#1397
Maybe the upstream pull request could implement a warning that there was no panel found? But then I'm not sure how we would ignore that warning...